src/sample/Controller/AlarmController.java

```diff
@@ -0,0 +1,24 @@
+package sample.Controller;
+
+
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+
+import java.io.File;
+
+public class AlarmController {
+    private MediaPlayer mediaPlayer;
+
+    public void playAlarm()
+    {
+       // System.err.println(getClass().getResource("/Users/monashamsolebad/sleepyhead/res/alarm.mp3").toString());
```

Comment:
What is this comment for?

Reply:
I used this line but I don't need it anymore